src/interfaces/SpokePool.ts

```diff
@@ -1,5 +1,21 @@
 import { SpokePoolClient } from "../clients";
+import { BigNumber } from "../utils";
 
 export interface SpokePoolClientsByChain {
   [chainId: number]: SpokePoolClient;
 }
+
+export interface ConvertedRelayData {
+  depositor: string;
+  recipient: string;
+  inputToken: string;
+  outputToken: string;
+  exclusiveRelayer: string;
+  originChainId: number;
+  depositId: BigNumber;
+  inputAmount: BigNumber;
+  outputAmount: BigNumber;
+  message: string;
+  fillDeadline: number;
+  exclusivityDeadline: number;
+}
```

Comment:
It's probably a good idea to derive this from the existing `RelayData`:

```suggestion
export interface ConvertedRelayData extends Omit<RelayData, "depositor" | "recipient" | "inputToken" | "outputToken" | "exclusiveRelayer" & {
  depositor: string;
  recipient: string;
  inputToken: string;
  outputToken: string;
  exclusiveRelayer: string;
}
```

Reply:
https://github.com/across-protocol/relayer/pull/2328/commits/a97b2b07487bba24fe4f704e2e53b09b87b5e197